src/renderer.ts

```diff
@@ -78,6 +77,7 @@ const Bullet = (props: { pillar: Pillar; text: string }): ReactElement =>
     h(Fragment, null,
         styledH('span', { css: bulletStyles(getPillarStyles(props.pillar).kicker) }, '•'),
         props.text.replace(/•/, ''),
+        h('br', null, null),
```

Comment:
Bullets would be inline without this.

Reply:
Can we solve that by adding a `display: block` to the `bulletStyles`?

Reply:
Tried that but then we'd have:

•
Some text
•
Some text

Reply:
Ah yeah, because those styles are on the `span`. In that case does the bullet actually make more semantic sense as a `<p>` tag?

```ts
const Bullet = (props: { pillar: Pillar; text: string }): ReactElement =>
    h('p', null,
        styledH('span', { css: bulletStyles(getPillarStyles(props.pillar).kicker) }, '•'),
        props.text.replace(/•/, ''),
    );
```